k8s/jenkins.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: extensions/v1beta1
```

Comment:
Pure ignorance on my part. What does `k8s` stand for?

Reply:
Kubernetes

Reply:
👍